locales/en/translation.json

```diff
@@ -457,13 +457,13 @@
   "This section requires data from an insulin pump, so there's nothing to display.": "",
   "This suspend happened because of one of the following alarms - no power, occlusion, auto-off, or no insulin.": "",
   "this year": "",
-  "Tidepool Applications Terms of Use": "",
-  "Tidepool is stuck and isn't doing what you want it to do. We're sorry for the trouble.": "",
-  "Tidepool provides free, secure data storage for diabetes data.": "",
+  "Tidepool Applications Terms of Use": "YourLoops Applications Terms of Use",
```

Comment:
Vous les avez rédigés / adaptés les terms of use ?

Reply:
Dans le backlog et en attente pour savoir si inclus avec DBLG1